pnpm-lock.yaml

```diff
@@ -220,7 +220,7 @@ importers:
         specifier: ^1.1.2
         version: 1.1.2
       '@qonto/ember-lottie':
-        specifier: 0.6.0
+        specifier: 0.6.1
```

Comment:
Is this `@qonto/ember-lottie` update wanted?

Reply:
Oh! Indeed it is 🤔 https://github.com/qonto/ember-lottie/blob/main/test-app/package.json#L40

Alright, nice catch 🙏